### PR TITLE
#1 Combined Turbo BYOP and Google Voice BYOP extensions

### PR DESCRIPTION
**Pull request by lcgrz**

### CODE REVIEW

**Comment by lcgrz** [comment]
@sttyerase mostly just tagged you on this as an FYI.  this really needs a lot more work before it is ready for primetime.

**Comment by sttyerase** [comment]
I suspected that a lot of unused and redundant code existed between the two modules.  I had not yet definitively identified what it was.  Let me know when you achieve a useable version and if I can help test it.

**Comment by sttyerase** [comment]
I gave it a quick test.  It sent the message twice(?). 
Screenshot 2023-08-17 at 11 48 29 PM

**Comment by lcgrz** [comment]
> I gave it a quick test. It sent the message twice(?). Screenshot 2023-08-17 at 11 48 29 PM

pretty sure that's a red herring.  i got this once yesterday sending a message manually (not via the extension).

**Comment by sttyerase** [comment]
Do you think it's merely a display issue?  I only gave it one quick test, but I'm pretty sure I saw the phone number entered twice as well. I will find some time to run more iterations.

**Comment by lcgrz** [comment]
> Do you think it's merely a display issue? I only gave it one quick test, but I'm pretty sure I saw the phone number entered twice as well. I will find some time to run more iterations.

honestly, i think it's a google issue, and i'm not going to worry myself over their product issues because there's nothing i can do about it.

File: background.js [hunk 1]
```diff
@@ -364,60 +68,53 @@ async function disableOrigin(origin) {
...
-        sessionRecords[sessionId][callNumber] = []
-    }
-    sessionRecords[sessionId][callNumber][TIMESTAMP_INDEX] = timestamp
-    sessionRecords[sessionId][callNumber][DURATION_INDEX] = duration
-    totalCalls += 1
+// For logging
+chrome.runtime.onMessage.addListener(function (message, sender, response) {
+    console.log('message', message);
 
-    // TODO make sure we don't run out of storage space
```
Comment (lcgrz) [inline background.js:381]: may want to restore totalCalls count.

**Review by sttyerase — APPROVED** [review]
Looks good.  The extension is working for all users so far.  This branch is ready to merge with main.  We can implement any needed changes with a new pull request.

**Comment by sttyerase** [comment]
Apparently I'm not an approved reviewer.  Git still seems to need a review to merge the code to main.

**Pull request merged**